### PR TITLE
Build wheels in Travis

### PR DESCRIPTION
**Pull request by kennydo**
Per https://github.com/quora/asynq/issues/66, it'd be useful to provide pre-built wheels to users of this library.

I mostly copied from `python-manylinux-demo`: https://github.com/pypa/python-manylinux-demo/blob/master/travis/build-wheels.sh. This change makes it so that travis pulls the manylinux Docker image, builds and tests inside there, and then the built artifacts are available in a directory that is created in the current working directory (`/io/wheelhouse` in the Docker image, just `wheelhouse` from the perspective of Travis).

A notable change I did make was removing python 3.3 from the Travis build matrix because the manylinux Docker image does not support it. Is Python 3.3 support worth special casing it in the Travis environment?

Unlike Circle CI, which makes test artifacts readily available for download, it looks like we have to manually tell Travis to upload these things somewhere (https://docs.travis-ci.com/user/uploading-artifacts/). So the maintainers of this package would still need to figure out how they want to deploy these files.

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
Thanks for doing this! We don't care about 3.3

**Comment by kennydo** [comment]
No problem! I'll also remove Python 3.3 from the setup.py classifiers then

**Comment by kennydo** [comment]
Github says "Only those with write access to this repository can merge pull requests", so whenever you're ready!

**Comment by JelleZijlstra** [comment]
We'll need to figure out too how to get Travis to upload the wheels to PyPI; I'll work on that soon.

**Comment by JelleZijlstra** [comment]
I'd like to move forward by making this change in github.com/quora/qcore first (that's a dependency of asynq, so wheels for asynq wouldn't be all that useful without it). I'll also want to configure it to have Travis upload releases on tag changes, but haven't figured out how to do that yet.

**Comment by kennydo** [comment]
Good point, I've created a PR for making the change to qcore: https://github.com/quora/qcore/pull/47

File: travis/build-wheels.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+PYBIN=/opt/python/$PYVER/bin
+
+"${PYBIN}/pip" install --upgrade setuptools  # work around https://github.com/pypa/setuptools/issues/1086
+
+# Compile wheels
+"${PYBIN}/pip" install -r /io/requirements.txt
+
+# Install dependencies of asynq
+"${PYBIN}/pip" install -U Cython qcore tox
```
Comment (JelleZijlstra) [inline travis/build-wheels.sh]: We should just add these to requirements.txt instead (not sure why we didn't do that before).

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+matrix:
+  include:
+    - python: "2.7"
+      env: PYVER=cp27-cp27m
+    - python: "3.4"
+      env: PYVER=cp34-cp34m
+    - python: "3.5"
+      env: PYVER=cp35-cp35m
+    - python: "3.6"
+      env: PYVER=cp36-cp36m
```
Comment (JelleZijlstra) [inline .travis.yml]: We should add 3.7 here like we do for qcore now.

Comment (JelleZijlstra) [reply .travis.yml]: (Made this change myself.)

**Pull request merged**